lib/execjs/runtimes.rb

```diff
@@ -89,6 +96,7 @@ def self.runtimes
         Duktape,
         MiniRacer,
         Node,
+        Bun,
```

Comment:
We can probably give it priority over node. Bun users are likely to also have node around. The inverse is less likely.

Reply:
On it!

Reply:
Done.